frontend/src/queries/schema/schema-general.ts

```diff
@@ -659,9 +664,10 @@ export interface DataTableNode
     hiddenColumns?: HogQLExpression[]
 }
 
-export interface GoalLine {
+export interface ChartAnnotation {
```

Comment:
There's a different usage of the word Annotation in charts, we use it in trends for github PR merges. (this thing)
![Uploading Screenshot 2025-01-15 at 18.49.59.png…]()

Please use a different name :D or keep to GoalLine

Reply:
Ah lol, sure. It sucks that `chart.js` calls it annotations though. Will revert to `GoalLine` :)